ingestion-data/staging/dataset-config/test-added.json

```diff
@@ -0,0 +1,78 @@
+{
+    "collection": "campfire-nlcd-TEST-ADDED",
+    "title": "Camp Fire Domain: Land Cover TEST ADDED",
+    "description": "NLCD-derived land use-land cover for 2016 over the Camp Fire burn scar domain. Displays colors 0-255 from a lookup table for land cover type (ex: reds = urbanized).",
+    "type": "Collection",
+    "spatial_extent": {
+        "xmin": -125,
+        "ymin": 40,
+        "xmax": -122,
+        "ymax": 45
+    },
+    "temporal_extent": {
+        "startdate": "2016-01-01T00:00:00Z",
+        "enddate": "2019-12-31T00:00:00Z"
+    },
+    "license": "CC0-1.0",
+    "is_periodic": false,
+    "time_density": "year",
```

Comment:
```suggestion

```
nit, but these are already set below with `dashboard:`. I think we probably have a variety of with prefix and without in this repo